src/main/java/io/appform/dropwizard/actors/actor/BaseActor.java

```diff
@@ -137,7 +137,11 @@ public final void publishWithDelay(final Message message, final long delayMillis
     public final void publishWithExpiry(final Message message, final long expiryInMs) throws Exception {
         actorImpl.publishWithExpiry(message, expiryInMs);
     }
-
+    public final void publishWithDelayAndExpiry(final Message message,
```

Comment:
Format code properly .. a line between functions. Fix this everywhere

Reply:
Add tests

Reply:
done